modules/apps/document-library/document-library-taglib/src/main/java/com/liferay/document/library/taglib/internal/servlet/RepositoryBrowserServlet.java

```diff
@@ -224,6 +226,12 @@ protected void doPut(
 			_sendResponse(httpServletResponse, HttpServletResponse.SC_OK);
 		}
 		catch (PortalException portalException) {
+			if (portalException instanceof DuplicateFolderNameException) {
+				_sendError(httpServletResponse, portalException.getMessage());
+
+				return;
+			}
+
```

Comment:
Can you split this in two distinct catch clauses?
```
catch (DuplicateFolderNameException duplicateFolderNameException) {
   _sendError(...);
}
catch  (PortalException portalException) {
   ...
}
```

Reply:
@adolfopa 
I updated source, thank you!